### PR TITLE
Go modules

### PR DESCRIPTION
**Pull request by vrothberg**
It's time. @mheon @baude @TomSweeneyRedHat @rhatdan PTAL

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
Fingers crossed for the tests, LGTM.

File: dependencies/dependencies.go [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package dependencies
+
+import (
+	_ "github.com/onsi/ginkgo/ginkgo"
+	_ "github.com/varlink/go/cmd/varlink-go-interface-generator" // Note: this file is used to trick `go mod` into vendoring the command below.
```
Comment (mheon) [inline dependencies/dependencies.go]: How did we never think of this before? This is actually brilliant

**Comment by mheon** [comment]
+100k -50k

Certainly starting the week off on a high note.

Regardless, this LGTM, but I'll let @baude have a look before we merge.

**Pull request merged**